README.md

```diff
@@ -52,6 +52,11 @@ docker run -d --name cassandra cassandra
 
 and in a while you should have Cassandra ready at port 9042. When the container has started, it's time to create a keyspace and a table for our data.
 
+Depending on the setup that you have you have you might want to bind the container directly to port 9042:
```

Comment:
I tried the commands in the readme and I just couldn't connect untill I bound it directly like this :(